rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1302,8 +1302,10 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 			return CELL_EIO; // ???
 		}
 
+		const fs::cell_device_stat hdd_info(info);
+
 		arg->out_block_size = mp->block_size;
-		arg->out_block_count = info.avail_free / mp->block_size;
+		arg->out_block_count = hdd_info.avail_free / mp->block_size;
```

Comment:
I think it reported 40mb here.

Reply:
"it"?

Reply:
The code?

Reply:
I don't care anymore. Feel free to fix all the games on your own